nikola/nikola.py

```diff
@@ -439,6 +439,7 @@ def __init__(self, **config):
             'BLOG_DESCRIPTION': 'Default Description',
             'BODY_END': "",
             'CACHE_FOLDER': 'cache',
+            'CATEGORIES_INDEX_PATH': '',
```

Comment:
Yes, that's the default value. It is the same as for `TAGS_INDEX_PATH`: empty string means that you want `output / TRANSLATION[lang] / CATEGORY_PATH / index.html` to be the path.